vmc/resource_vmc_sddc_test.go

```diff
@@ -133,6 +133,7 @@ resource "vmc_sddc" "sddc_1" {
 	delay_account_link  = false
 	skip_creating_vxlan = false
 	sso_domain          = "vmc.local"
+    size = "large"
```

Comment:
is the indentation correct?

Reply:
Indentation seems off here but if you open the file it seems to be okay.

Reply:
ok